src/services/url.ts

```diff
@@ -1,4 +1,5 @@
-import { Events, Selectors, Store } from '@storefront/flux-capacitor';
+import { Events, Routes, Selectors, Store } from '@storefront/flux-capacitor';
+import * as uuid from 'uuid';
```

Comment:
I feel like we don't use `uuid` anymore here so you can probably remove it as a dependency and anywhere it's referenced